sql/structured.go

```diff
@@ -42,11 +45,71 @@ const (
 	MaxReservedDescID ID = 999
 	// RootNamespaceID is the ID of the root namespace.
 	RootNamespaceID ID = 0
+
+	// systemDatabaseID is the ID of the system database.
+	systemDatabaseID ID = 1
+	// namespaceTableID is the ID of the namespace table.
+	namespaceTableID ID = 2
+	// descriptorTableID is the ID of the descriptor table.
+	descriptorTableID ID = 3
 )
 
 // ErrMissingPrimaryKey exported to the sql package.
 var ErrMissingPrimaryKey = errors.New("table must contain a primary key")
 
+// SystemDB is the descriptor for the system database.
+var SystemDB = DatabaseDescriptor{
+	Name:       "system",
+	ID:         systemDatabaseID,
+	Privileges: NewDefaultDatabasePrivilegeDescriptor(),
```

Comment:
yup. The only thing preventing it right now is the `PrivilegeDescriptor.Validate()` method, but that can easily change.
We should special-case it such that only `SELECT` is allowed. It's fine to grant read-access to other people, but even `root` shouldn't write to it through the sql interface.